plugins/coverage/token.js

```diff
@@ -18,32 +18,21 @@ module.exports = config => ({
                 security: [{ token: [] }]
             }
         },
-        handler: (request, reply) => {
+        handler: async (request, reply) => {
             const buildCredentials = request.auth.credentials;
-            const { jobFactory } = request.server.app;
             const { jobId } = buildCredentials;
```

Comment:
`buildCredentials` contains `prParentJobId` & `isPR`, you don't need to fetch job again in `getCoverageConfig` for them